pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/BarnepensjonForeldreloesFraser.kt

```diff
@@ -49,6 +56,16 @@ object BarnepensjonForeldreloesFraser {
                         Language.English to "We refer to the advance notice about a new children’s pension scheme. You have been granted a children's pension again from ".expr() +
                                 formatertVirkningsdato + " both your parents are registered as deceased."
                     )
+                }.orIfNotNull(forskjelligAvdoedPeriode) { forskjelligAvdoed ->
+                    val avdoedNavn = forskjelligAvdoed.foersteAvdoed.navn
+                    val formatertDoedsdato = forskjelligAvdoed.foersteAvdoed.doedsdato.format()
+                    val senereDoedsdato = forskjelligAvdoed.senereAvdoed.doedsdato.format()
+                    val senereVirkningsdato = forskjelligAvdoed.senereVirkningsdato.format()
+                    textExpr(
+                        Language.Bokmal to "Du er innvilget barnepensjon fra ".expr() + formatertVirkningsdato + " fordi " + avdoedNavn + " er registrert død " + formatertDoedsdato + ". Barnepensjon endres fra " + senereVirkningsdato + " fordi den andre forelderen din er registrert død " + senereDoedsdato + ". ",
```

Comment:
Muligens jeg misser på noe, men hvorfor refereres det til EN foreldre på bokmål "den andre foreldren din", mens for  Nynorsk og Engelsk refereres det til BEGGE "fordi begge foreldra dine".